src/kvtable.rs

```diff
@@ -70,7 +70,7 @@ impl<K, V> Drop for KVs<K, V> {
 }
 
 // ---Structure for resizing -------------------------------------------------------
-
+#[derive(Default)]
```

Comment:
I think it's fine not to implement `Default` for `CHM` as it should not be made public anyway. I'll change its visibility later.